### ISSUE TITLE
ParameterRuns.jl issue

### ISSUE THREAD

**Issue by alee** [comses-education/spatialrust-model#3]
It looks like there's an issue with the ParameterRuns.jl file, `shading_experiment not defined` - I can run the OneRun.jl fine though on OSG - thanks @manuvanegas !

```
Start time: 2022-06-24T12:58-05:00
Job running on node: c1506.swan.hcc.unl.edu
OSG site: SwanJob running as user: uid=32016(osg) gid=4001(grid) groups=4001(grid)
Command line args: /srv/ParameterRuns.jl  Activating project at `/code`
ERROR: LoadError: UndefVarError: shading_experiment not defined
Stacktrace:
 [1] top-level scope
   @ ~/ParameterRuns.jl:41
in expression starting at /srv/ParameterRuns.jl:41

Update message: DrWatson v2.7.3

* New section "Taking project input-output automation to 11" in the documentation.
  It showcases how to eliminate code duplication and streamline your simulation setup
  and run phase using `savename` and `produce_or_load`.
* By default now `gitpatch` is NOT saved when calling `tag!` and derivative functions.
  This is due to an unknown problem that causes collecting the git patch to
  never halt, potentially not saving a user's output.
```

**Comment by manuvanegas** [comment]
Sorry! I was going to push the fix when I saw that you already got it.

**Comment by alee** [comment]
fixed in d4a272b09a1bff1d1984abe668112ba58ca1be9a - no problem @manuvanegas ! I just poked around and saw it was a simple name change :sweat_smile:
